apps/pie-docs/src/engineers/getting-started/structure.md

```diff
@@ -49,6 +50,93 @@ For more details on Fozzie, Snacks and Skip PIE, please check out the **document
 
 ---
 
+## What library to use?
+
+### Latest tech stack and Vanilla JS
+
+{% contentLayout %}
+    {% tag {
+    label: "Next 14",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "React 18",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Nuxt 3",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Vue 3",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Vanilla JS",
+    variant: "blue"
+    } %}
+{% endcontentLayout %}
+
+If you are working with latest stack both in React and Vue ecosystems, we recommend to go straight away with
+[PIE Web Components](/engineers/web-components/)
+
+### Next 13 and below 
+
+{% contentLayout %}
+    {% tag {
+    label: "Next 13",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Next 10",
+    variant: "blue"
+    } %}
+{% endcontentLayout %}
+ 
+For Next 13 and below, we recommend using [PIE Web Components](/engineers/web-components/)
+only for client side components. If you need to render components on the server (SSR), [Snacks](https://snacks.takeaway.com/portal/) is the best library for you.
+
+### Vue and Nuxt 2
+
+{% contentLayout %}
+    {% tag {
+    label: "Vue 2",
+    variant: "blue"
+    } %}
+    {% tag {
+    label: "Nuxt 2",
+    variant: "blue"
+    } %}
+{% endcontentLayout %}
+
+Our [Fozzie](https://vue.pie.design/) library currently supports older versions of Vue. We recommend updating to the latest Vue ecosystem if possible.
+
+### Is your project a greenfield?
+
+We recommend using [PIE Web Components](/engineers/web-components/) as much as you can, if your tech stack is supported. This helps us test our new components and avoids any need for a future migration.
+
+### I can't find the web component I need
+
+**Do you need it right away?**
+
+If you need the component right away, please check our legacy systems ([Snacks](https://snacks.takeaway.com/), [Fozzie](https://vue.pie.design/)) in case a similar component exist.
```

Comment:
```suggestion
If you need the component right away, please check our legacy libraries ([Snacks](https://snacks.takeaway.com/) and [Fozzie](https://vue.pie.design/)) in case a suitable component already exists.
```